src/main/java/com/school/project/model/entities/Lesson.java

```diff
@@ -8,29 +8,34 @@
 
 import javax.persistence.*;
 import java.util.Date;
+import java.util.List;
 
 @Entity
 @Table(name = "LESSON")
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
-
 public class Lesson {
 
     @Id
     @Column(name = "ID")
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
     @Column(name = "THEMA", nullable = false)
     private String thema;
-    @ManyToOne
-    @JoinColumn(name = "SUBJECT_ID", nullable = false)
-    private Subject subject;
+
+    @OneToMany(targetEntity = Subject.class)
+    @JoinTable(name = "LESSON_SUBJECT", joinColumns = {@JoinColumn(name = "LESSON_ID")},
+            inverseJoinColumns = {@JoinColumn(name = "SUBJECT_ID")})
+    private List<Subject> subjects;
+
     @CreationTimestamp
     @Temporal(TemporalType.TIMESTAMP)
     @Column(name = "LESSON_DATE", nullable = false)
```

Comment:
pls rename to CREATED_DATE and the name of property also from date to creationDate